src/redux/actions/cellSets/runCellSetsClustering.js

```diff
@@ -21,7 +21,7 @@ const runCellSetsClustering = (experimentId, resolution) => async (dispatch, get
   const body = {
     name: 'ClusterCells',
     cellSetName: 'Louvain clusters',
-    type: 'louvain',
```

Comment:
nice one! Very good to be removing all those hardcodings of louvain, because very soon we want to be able to support other clustering methods, not only louvain.